src/main/java/uk/gov/companieshouse/api/accounts/controller/CompanyAccountController.java

```diff
@@ -71,7 +90,8 @@ public ResponseEntity getCompanyAccount(HttpServletRequest request) {
             return ResponseEntity.status(HttpServletResponse.SC_NOT_FOUND).body(null);
         }
 
-        return ResponseEntity.status(HttpStatus.OK).body(companyAccountTransformer.transform(companyAccountEntity));
+        return ResponseEntity.status(HttpStatus.OK)
```

Comment:
I know this is only changed for splitting the line but have to point out that this code needs to be updated to handle errors in the transform method i.e. if the transform fails we should detect that and deal with it appropriately with a useful log message and response code